README.md

```diff
@@ -86,9 +86,23 @@ Unix tools:
 
 [Google Chrome]: https://www.google.com/chrome/
 
+# iTerm2
```

Comment:
More opinionated / personal so perhaps drop :)

Reply:
ok but if not then Karabiner? remapping the keyboard shouldn't be impossible, otherwise you have a non-functional home/end keys